automatic/sonarlint-vs2019/ReadMe.md

```diff
@@ -0,0 +1,42 @@
+﻿# <img src="https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages/tree/master/icons/sonarlint-vs2019.png" width="48" height="48"/> [SonarLint for Visual Studio 2019](https://chocolatey.org/packages/sonarlint-vs2019)
+
+## Overview
```

Comment:
Follow [Core Team Package Repo Guidelines](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description) regarding package description. You can also see other packages in this repo or [chocolatey-coreteampackages](https://github.com/chocolatey-community/chocolatey-coreteampackages) repo as examples.